tests/generators/test_multirest.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from garak.generators.rest import MultiRestGenerator
+from garak import _config, _plugins
+from pathlib import Path
+from garak.data import path as data_path
+            
+EXAMPLE_BURP_XML = {"burpfile": str(Path(data_path) / "rest" / "multirest.xml") }
```

Comment:
Sorry in tests this should be:
```suggestion
EXAMPLE_BURP_XML = {"burpfile": str(_config.transient.data_dir / "data" / "rest" / "multirest.xml") }
```

This was my oversight, I should not have recommended the `data_path` helper here.